client/apps/shipping-label/redux-middleware.js

```diff
@@ -20,7 +20,10 @@ const middlewareActions = {
 	},
 	[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_OPEN_PRINTING_FLOW ]: () => {
 		//dismiss the nux pointer if the user opens the printing flow
-		jQuery( '#woocommerce-order-label' ).pointer( 'close' );
+		const labelMetabox = jQuery( '#woocommerce-order-label' );
+		if ( labelMetabox.pointer ) {
+			labelMetabox.pointer().pointer( 'close' );
```

Comment:
Why did the call need to change here from `.pointer( 'close' )` to `.pointer().pointer( 'close' )` ?